ios/ImagePickerManager.m

```diff
@@ -241,15 +241,15 @@ -(NSMutableDictionary *)mapVideoToAsset:(NSURL *)url error:(NSError **)error {
         }
 
         if (url) { // Protect against reported crash
-
+            NSError * err = nil;
```

Comment:
`mapVideoToAsset:error` function already takes an NSError(error) as a param. We use this `error` in [FileManager...] call. So why do we require this change.

Reply:
The place we call `mapVideoToAsset:error` function expects the error object to be set if there is any error, here ur creating local variable which will not be reflected.

Reply:
Hi @ravirajn22 ,

Thanks for the prompt review. After debug again and I found out my fixes is not efficient. 

Actually I'm trying to bypass the error pointer passed from 
`- (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary<NSString *,id> *)info` to `mapVideoToAsset`, and only do checking for the FileManager error. 

The reason I do that is because I found out after done picking Video file from image picker, the error is nil. But after passed to 
`mapVideoToAsset` the error pointer is holding something. And end up returned nil error value for callback.

You can refer to my video for more details.

https://user-images.githubusercontent.com/22317616/121286443-6f5ea500-c912-11eb-8fd9-9beb6cc87fda.mov